src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/PE/LoadCustomModifiers.cs

```diff
@@ -250,6 +250,769 @@ public void TestFieldCustomModifierCount()
             Assert.Equal(0, CustomModifierCount(@class.GetMember<FieldSymbol>("field00")));
         }
 
+        [Fact]
+        public void RejectIsReadOnlySymbolsThatShouldHaveInAttributeModreqButDoNot_Delegates_Parameters()
```

Comment:
Consider shortening the test names. Perhaps `MissingInAttributeModreq_Delegates_Parameters`